### ISSUE TITLE
Selectively updating database?

### ISSUE THREAD

**Issue by Futureknows** [cboxdoerfer/fsearch#138]
I have several large external HDDs that take time to index but I rarely need to index. However everytime I want to update the index of my SSD, I have to wait for fsearch to do the whole shebang. Is it possible to set fsearch to only reindex the SSD (near instantaneous), and then only index the HDDs on demand?

**Comment by cboxdoerfer** [comment]
Yes, doing selective updates will be possible. The next big update of fsearch includes a small binary which only handles database updates, and you can easily tell it to only update certain paths.

**Comment by apapapapapapap** [comment]
That would be a huge increase of fsearch's usability (at least for me). Since i have several harddisks with huge amounts of small files a fsearch database-update takes many minutes. to be able to schedule that and/or to include/exclude certain disks would be perfect. Many thanks for your work so far.

**Comment by cs96and** [comment]
#165

**Comment by cs96and** [comment]
#166

**Comment by cboxdoerfer** [comment]
While it's not exactly what was asked for, I just added the ability to disable updates of individual paths. Searching in those will still be possible, if FSearch was able to at least scan them once and write their database content to disk. This way we can avoid rescans of large and slow HDDs, but still keep other paths updated. Hopefully this makes FSearch at least a little more pleasant to use.

Of course, the next step will be the ability to update paths individually from each other, both from the command line and preferences dialog.
